core/ide/src/main/java/org/alice/stageide/sceneeditor/StorytellingSceneEditor.java

```diff
@@ -477,38 +477,33 @@ public void setSelectedExpression(Expression expression) {
 
   private void updateOrthographicCameraView(EntityImp targetImplementation) {
     double targetHeight = targetImplementation.getAxisAlignedMinimumBoundingBox().getHeight();
+    double targetWidth = targetImplementation.getAxisAlignedMinimumBoundingBox().getWidth();
+    double targetDepth = targetImplementation.getAxisAlignedMinimumBoundingBox().getDepth();
+    double defaultDistance = 10.0f;
     AffineMatrix4x4 targetTransform = targetImplementation.getAbsoluteTransformation();
-    ClippedZPlane picturePlane = new ClippedZPlane();
 
     AffineMatrix4x4 topTransform = AffineMatrix4x4.createIdentity();
     topTransform.translation.x = targetTransform.translation.x;
-    topTransform.translation.y = 10;
+    topTransform.translation.y = targetHeight != 0 ? targetHeight * 10 : defaultDistance;
     topTransform.translation.z = targetTransform.translation.z;
     topTransform.orientation.up.set(0, 0, 1);
     topTransform.orientation.right.set(-1, 0, 0);
     topTransform.orientation.backward.set(0, 1, 0);
     this.topOrthoMarkerImp.setLocalTransformation(topTransform);
-    picturePlane.setCenter(0, 0);
```

Comment:
I steered you wrong in removing these. The in/out arrows affect this, not the position of the camera, so using these arrows lead to persistent effects across views.
All three should setPicturePlane calls be restored and amended.
Instead of using fixed values, set them based on the model, perhaps plane.setHeight(2 * targetHeight)
From above the depth of the object corresponds to the height of the windows, so use that here.